bftengine/src/bftengine/SigManager.hpp

```diff
@@ -55,6 +55,7 @@ class SigManager {
                           concord::crypto::KeyFormat replicasKeysFormat,
                           const std::set<std::pair<const std::string, std::set<uint16_t>>>* publicKeysOfClients,
                           concord::crypto::KeyFormat clientsKeysFormat,
+                          const std::tuple<PrincipalId, Key, concord::crypto::KeyFormat>& operatorKey,
```

Comment:
Since clients also use sigmanager, the interface should allow to add an operator if wanted, but have a null default (std::optional supports this behavior).

Reply:
done